ci/azure-pipelines.yml

```diff
@@ -50,8 +50,7 @@ jobs:
     steps:
       - template: azp-dependencies.yml
       - checkout: self
-      - script: make unit-test-wasm
-        displayName: Run unit test wasm
+      - bash: echo "no unit-test-wasm"
```

Comment:
We do not have `unit-test-wasm`. We need to remove it from `required`  then I will remove the job.